src/applications/vaos/services/location/transformers.v2.js

```diff
@@ -32,7 +60,39 @@ export function transformParentFacilityV2(parentFacility) {
 export function transformParentFacilitiesV2(facilities) {
   return facilities
     .filter(facility => {
-      return facility.id === facility.vast_parent;
+      return facility.id === facility.vastParent;
     })
-    .map(transformParentFacilityV2);
+    .map(transformFacility);
+}
+
+/**
+ * Transforms a list of MFS v2 facilities into a list of Locations
+ *
+ * @export
+ * @param {Array<MFSFacility>} facilities A list of facilities from MFS v2 facilities endpoint
+ * @returns {Array<Location>} A list of Location resources
+ */
+export function transformFacilitiesV2(facilities) {
+  return facilities.map(transformFacility);
+}
+
+function getTypeOfCareIdFromV2(id) {
+  const allTypesOfCare = [
+    ...TYPES_OF_EYE_CARE,
+    ...TYPES_OF_SLEEP_CARE,
+    ...AUDIOLOGY_TYPES_OF_CARE,
+    ...TYPES_OF_CARE,
+  ];
+
+  return allTypesOfCare.find(care => care.idV2 === id)?.id;
+}
+
+export function transformSettingsV2(settings) {
+  return settings.map(setting => ({
+    ...setting,
+    services: setting.services.map(service => ({
+      ...service,
+      id: getTypeOfCareIdFromV2(service.id),
```

Comment:
We're converting the V2 ids to our old ids for now, will probably change this around in the future.